config/routes.rb

```diff
@@ -4,23 +4,34 @@
 
   # You can have the root of your site routed with "root"
   root 'tasks#index'
-  post '/' => 'tasks#index'
-
   # Example of regular route:
-  get 'show/:id' => 'tasks#show'
+  get 'tasks/show/:id' => 'tasks#show'
```

Comment:
Per the RESTful routes definitions, we'd generally put the :id between the controller and corresponding action, so `tasks/:id/complete`.

I think based on how much more you've used RESTful routing since this project, I'm sure you'd have ideas on some other things that you'd want to change.